Assets/Scripts/Device/GlobalControls.cs

```diff
@@ -110,14 +110,13 @@ public class GlobalControls : MonoBehaviour {
                     else if (Input.anyKeyDown)       fleeIndex = 0;
                     break;
             }
-		if (!Screen.fullScreen && (Screen.currentResolution.height != 480 || Screen.currentResolution.width != 640)) {
+        if (!Screen.fullScreen && (Screen.currentResolution.height != 480 || Screen.currentResolution.width != 640)) {
```

Comment:
If that block is a one line long if block, you can safely remove the brackets :P